testnet/stacks-node/src/nakamoto_node/miner.rs

```diff
@@ -827,6 +830,7 @@ impl BlockMinerThread {
             //  correct signer_sighash for `process_mined_nakamoto_block_event`
             Some(&self.event_dispatcher),
             signer_transactions,
+            signer_bitvec_len.unwrap_or(1),
```

Comment:
Why 1 and not 0?

Reply:
I don't have a good argument - mainly `NakamotoBlock::empty()` and `NakamotoBlock::genesis()` use 1, so I went with that.

Reply:
Changed this to be zero